src/waveform/visualplayposition.h

```diff
@@ -60,12 +60,12 @@ class VisualPlayPosition : public QObject {
     void setInvalid() { m_valid = false; };
 
   private slots:
-    void slotAudioBufferSizeChanged(double size);
+    void slotAudioBufferSizeChanged(double sizeMs);
 
   private:
     ControlValueAtomic<VisualPlayPositionData> m_data;
     ControlProxy* m_audioBufferSize;
-    double m_dAudioBufferSize; // Audio buffer size in ms
+    double m_dAudioBufferUs; // Audio buffer size in µs
```

Comment:
"Us" is very confusing, used here as both a suffix and a prefix. Legacy code, I know. The code would be easier to read if we consequently explicitly append "Millis" and "Micros" as a suffix for all durations that are used here. With this convention even the trivial constant kMicrosPerMilli would be obsolete, because then the factor 1000 is obvious and even more concise.

Labelling a duration as "size" is misleading too, but let's just keep it consistent ;)